jetstream/util.py

```diff
@@ -0,0 +1,15 @@
+from contextlib import contextmanager
+from pathlib import Path
+import shutil
+import tempfile
+
+# based on https://stackoverflow.com/a/22726782
+
+
+@contextmanager
+def TemporaryDirectory():
+    name = Path(tempfile.mkdtemp())
+    try:
+        yield name
+    finally:
+        shutil.rmtree(name)
```

Comment:
This tries to address https://github.com/mozilla/jetstream/pull/283#discussion_r505693169